data-processing/common/commands/database.py

```diff
@@ -14,7 +14,7 @@ class DatabaseUploadCommand(ArxivBatchCommand[I, R], ABC):
 
     def __init__(self, args: Any) -> None:
         super().__init__(args)
-        init_database_connections(schema_name=args.schema, create_tables=True)
+        init_database_connections(schema_name=args.schema, create_tables=False)
```

Comment:
I do like the ability to create tables by default if someone is uploading data to a short-lived test schema. Is there some way to expose the ability to create tables through command line arguments if we're going to disable default creation of tables?

Reply:
I added `--database-create-tables` flag to address this. By default it's False b/c without more changes providing True requires admin permissions on the DB database.